indico/web/static/images/src/social.svg

```diff
@@ -1,1164 +1,134 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Is the old social icon still used anywhere? Maybe we could just get rid of it?

Same goes for the styles related to the old share widget:

https://github.com/indico/indico/blob/aa73eb0da6c632952bfdbeabf69570ad9ce021b0/indico/web/client/styles/partials/_footer.scss#L57-L69